### PR TITLE
[NAE-1721] Create case event bad handling on frontend

### PR DESCRIPTION
**Pull request by Kovy95**
# Description
- handle error

Implements [NAE-1721]

## Dependencies
none

### Third party dependencies
No new dependencies were introduced

### Blocking Pull requests
There are no dependencies on other PR

## How Has Been This Tested?
manually

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |     Linux Mint      |
| Runtime             |     NodeJS     |
| Dependency Manager  |     NPM      |
| Framework version   |     Andular 13.2      |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My changes have been checked, personally or remotely, with @...
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_components)
    - [x] [Snyk](https://app.snyk.io/org/netgrif)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1721]: https://netgrif.atlassian.net/browse/NAE-1721?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/netgrif-components-core/src/assets/i18n/en.json [hunk 1]
```diff
@@ -87,7 +87,8 @@
             "errThird": "Color is required.",
             "noNets": "No allowed Nets",
             "createCase": "Successful create new case",
-            "defaultCaseName": "with default case name"
+            "defaultCaseName": "with default case name",
+            "createCaseError": "New case created, but an error occurred while executing actions"
```
Comment (machacjozef) [inline projects/netgrif-components-core/src/assets/i18n/en.json:91]: New case was created, but an error occurred while executing actions

maybe: 
A new case was created, but an error occurred while executing actions


by @minop

File: projects/netgrif-components-core/src/assets/i18n/de.json [hunk 2]
```diff
@@ -87,7 +87,8 @@
             "errThird": "Sie müssen eine Farbe auswählen.",
             "noNets": "Es gibt keine erlaubte Netze",
             "createCase": "Neuen Fall würde erfolgreich erzeugt",
-            "defaultCaseName": "mit einem Standardfallnamen"
+            "defaultCaseName": "mit einem Standardfallnamen",
+            "createCaseError": "Neuer Fall erstellt, aber beim Ausführen von Aktionen ist ein Fehler aufgetreten"
```
Comment (machacjozef) [inline projects/netgrif-components-core/src/assets/i18n/de.json:91]: Neuer Fall wurde erstellt, aber beim Ausführen von Aktionen ist ein Fehler aufgetreten



by @minop

File: projects/netgrif-components-core/src/assets/i18n/sk.json [hunk 3]
```diff
@@ -87,7 +87,8 @@
             "errThird": "Farba je povinné pole.",
             "noNets": "Žiadne povolené siete",
             "createCase": "Úspešne vytvorený nový prípad",
-            "defaultCaseName": "s prednastaveným názvom prípadu"
+            "defaultCaseName": "s prednastaveným názvom prípadu",
+            "createCaseError": "Nový prípad vytvorený, nastala však chyba pri vykonávanií akcií"
```
Comment (minop) [inline projects/netgrif-components-core/src/assets/i18n/sk.json:91]: vykonávaní

**Review by minop — CHANGES_REQUESTED** [review]

File: projects/netgrif-components-core/src/assets/i18n/sk.json (hunk 3 above)
Comment (Kovy95) [reply projects/netgrif-components-core/src/assets/i18n/sk.json]: done

**Review by minop — APPROVED** [review]

**Review by mladoniczky — APPROVED** [review]

**Pull request merged**